src/Contract/Test/E2E/Helpers.purs

```diff
@@ -71,14 +75,17 @@ typeInto selector text page = toAffE $ _typeInto selector text page
 -- | Find the popup page of the wallet. This works for both Nami and Gero
 -- | by looking for a page with a button. If there is a button on the main page
 -- | this needs to be modified.
-findWalletPage :: ExtensionId -> Toppokki.Browser -> Aff (Maybe Toppokki.Page)
```

Comment:
It seems `ExtensionId` is no longer used, but the newtype is still defined - should it be deleted?

Reply:
It is used, e.g. in `inWalletPageOptional`